vagrant/ansible/roles/client.prep/tasks/main.yml

```diff
@@ -8,9 +8,18 @@
       - git
     state: latest
 
-- name: Install Python yaml package
+- name: Install RHEL 7 specific packages
+  yum:
+    name:
+      - python36-iso8601
+    state: latest
+  when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '7'
+
+- name: Install Python modules using pip
   pip:
-    name: PyYAML
+    name:
+      - PyYAML
+      - iso8601
```

Comment:
Does the `executable` [parameter](https://docs.ansible.com/ansible/latest/modules/pip_module.html#parameter-executable) help us here?

Reply:
If we do use it, we will end up with the when condition for the RHEL 7 specific install here. In any case, PyYAML is used by my testing script which uses the default python and iso8601 is used by the selftest environment.

I could in theory run my scripts with python 3 and force module install with pip3 instead.

Reply:
Resolved with the latest set of patches in PR#99, 100, 101.